lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -421,7 +421,7 @@ If ($state -eq "present") {
 	}
 
 	If ($insertbefore -and $insertafter) {
-		Add-Warning('Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"');
+		Add-Warning $result 'Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"'
```

Comment:
Are you able to change the message to be `"` double quotes. The variable `$insertafter` won't actually be shown in this example. Would also be good to use `-obj` and `-message` for the parameters.